analysis/study_definition_outcomes_1.py

```diff
@@ -0,0 +1,136 @@
+##############################################################################
+#
+# This script provides the formal specification of the study data
+# that will be extracted from
+# the OpenSAFELY database.
+#
+# STUDY PURPOSE: to perform regression discontinuity of 2022/23 
+#   autumn booster COVID-19 vaccine, before and after 50+ became eligible
+#   on October 15, 2022
+#
+# This study definition extracts information on outcomes for the control
+# periods - i.e. 28 days after Sep 03 (pre-campaign), 
+# and 28 days after Oct 15 (start of campaign). Only one outcome (first)
+# per person is extracted.
+#
+##############################################################################
+
+
+# IMPORT STATEMENTS ----
+
+# Import code building blocks from cohort extractor package
+from cohortextractor import (
+    StudyDefinition,
+    patients,
+    Measure,
+    codelist,
+)
+
+# Import codelists from codelist.py (which pulls them from the codelist folder)
+from codelists import *
+
+COHORT = "output/cohort/cohort_final_sep.csv"
+
+# Specify study definition
+study = StudyDefinition(
+
+    # Configure the expectations framework
+    default_expectations = {
+        "date": {"earliest": "2020-12-08", "latest": "2023-02-01"},
+        "rate": "uniform",
+        "incidence": 0.5,
+    },
+        
+    # Set index date
+    index_date = "2022-09-03",
```

Comment:
i know this index_date is overwritten by the `--index-date-range` option in the project.yaml, so it works. But you might consider using study def paramterisation for this instead of `--index-date-range`, just for clarity. Really not a big deal either way.